src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/bitbucket/ApiClient.java

```diff
@@ -193,6 +184,23 @@ public void updatePullRequestComment(String pullRequestId, String content, Strin
         return null;
     }
 
+    private <T> List<T> getAllValues(String rootUrl, int pageLen, Class<T> cls) {
+        List<T> values = new ArrayList<T>();
+        try {
+            String url = rootUrl + "?pagelen=" + pageLen;
```

Comment:
Nit: it might be more elegant to parse any existing query string from rootUrl and append the `pagelen` param to the map before reencoding it.